patches/minecraft/net/minecraft/entity/monster/ZombieEntity.java.patch

```diff
@@ -1,23 +1,24 @@
 --- a/net/minecraft/entity/monster/ZombieEntity.java
 +++ b/net/minecraft/entity/monster/ZombieEntity.java
-@@ -186,7 +186,7 @@
+@@ -186,7 +186,8 @@
        if (!this.field_70170_p.field_72995_K && this.func_70089_S() && !this.func_175446_cd()) {
           if (this.func_204706_dD()) {
              --this.field_204708_bE;
 -            if (this.field_204708_bE < 0) {
-+            if (this.field_204708_bE < 0 && net.minecraftforge.event.ForgeEventFactory.canLivingConvert(this, EntityType.field_200725_aD)) {
++
++            if (this.field_204708_bE < 0 && net.minecraftforge.event.ForgeEventFactory.canLivingConvert(this, EntityType.field_200725_aD, (timer) -> this.drownedConversionTime = timer)) {
```

Comment:
Something is broken here.. did you edit this file by hand? The field should of been mapped to the srg name. I mean.. it's fixed in the first part of the line, but the lamba is broken...